GitSearch/Models/SpinnerViewController.swift

```diff
@@ -24,3 +35,28 @@ class SpinnerViewController: UIViewController {
             ])
     }
 }
+
+extension SpinnerViewController {
+    
+    func toggle() {
```

Comment:
Probably I was wrong when I recommended for you to create `toggle` method. It looks somewhat incorrect in this scenario. I would suggest reimplementing `SpinnerViewController` functionality as an extension on `UIView` instead of custom view controller.